sonar-python-plugin/src/test/java/org/sonar/plugins/python/MockSonarLintIssue.java

```diff
@@ -98,8 +105,8 @@ public NewIssue addFlow(Iterable<NewIssueLocation> flowLocations) {
   }
 
   @Override
-  public NewIssue addFlow(Iterable<NewIssueLocation> flowLocations, FlowType flowType, @Nullable String flowDescription) {
-    throw new IllegalStateException("Not supposed to be tested");
+  public NewIssue addFlow(Iterable<NewIssueLocation> iterable, FlowType flowType, @Nullable String s) {
+    return null;
```

Comment:
is this change intentional? Why do we return null instead of throwing an exception?

Reply:
I just took the mock class implementation from sonar-java project.